symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -34,6 +45,8 @@
 @API(status = API.Status.STABLE)
 public class ApiClientBuilderJersey2 implements ApiClientBuilder {
 
+  private static final Logger logger = LoggerFactory.getLogger(ApiClientBuilderJersey2.class);
```

Comment:
don't we use @Slf4j (lombok) in the BDK?

Reply:
This is what used in ApiClientBuilderWebClient, should I change both of them to use lombok slf4j?